.gitignore

```diff
@@ -0,0 +1,9 @@
+
```

Comment:
Isn't there a standard .gitignore for Godot? :thinking:

Reply:
that is the standard one but i copy pasted it..... should i remove the newline?

Reply:
Yeah that first line can be removed :)